lib/controllers/wchamps.js

```diff
@@ -0,0 +1,8 @@
+const { Router } = require('express');
+const WorldChampsWomen = require('../models/WorldChampsWomen');
+
+module.exports = Router().get('/', async (req, res) => {
+  const wchamps = await WorldChampsWomen.getAll();
+  const filtered = wchamps.map(({ id, name }) => ({ id, name }));
```

Comment:
Nice job implementing `.map`, destructuring, AND using an implied return here. Looks great.